musicbrainzngs/musicbrainz.py

```diff
@@ -685,7 +685,7 @@ def _get_auth_type(entity, id, includes):
     """ Some calls require authentication. This returns
     True if a call does, False otherwise
```

Comment:
While we're playing around in this function perhaps we should update this doc, which seems to have been out of date for many years